components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -123,7 +146,11 @@ class MozillaSocorroService(
         val gzipOs = GZIPOutputStream(os)
         sendPart(gzipOs, boundary, "ProductName", appName, nameSet)
         sendPart(gzipOs, boundary, "ProductID", appId, nameSet)
-        sendPart(gzipOs, boundary, "Version", version, nameSet)
+        sendPart(gzipOs, boundary, "Version", versionName, nameSet)
+        sendPart(gzipOs, boundary, "AcVersion", AcBuild.version, nameSet)
+        sendPart(gzipOs, boundary, "GleanVersion", AcBuild.gleanSdkVersion, nameSet)
+        sendPart(gzipOs, boundary, "AsVersion", AcBuild.applicationServicesVersion, nameSet)
+        sendPart(gzipOs, boundary, "GeckoViewVersion", version, nameSet)
```

Comment:
Will those other versions get exposed in the crash-stats UI? :)

Reply:
Crash Stats shows fields that are explicitly marked public to everyone. Fields that aren't explicitly marked public only show up for people who have access to PII.

If you want additional fields to show up for everyone, be searchable or aggregatable, or something along those lines, please open up a bug against the Socorro product and we can work through the request.